ibm/service/cis/data_source_ibm_cis_mtls.go

```diff
@@ -0,0 +1,136 @@
+// Copyright IBM Corp. 2017, 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package cis
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	cisMtlsOutput                  = "mtls_certificates"
+	cisMtlsCertID                  = "cert_id"
```

Comment:
Why we need to have cert_XXXXX
can we have just "id", "name", "fingerprint"...."expires_on"